SCClassLibrary/Common/Control/Engine.sc

```diff
@@ -165,17 +165,19 @@ ContiguousBlock {
 	*new { |start, size| ^super.newCopyArgs(start, size) }
 
 	address { ^start }
+	nextAddress { ^start + size }
+	lastAddress { ^start + size - 1 }
```

Comment:
This reads much clearer.

I don't quite understand `lastAddress` yet – could you add documentation for the two new methods?

Reply:
> I don't quite understand lastAddress yet – could you add documentation for the two new methods?

Done.

(BTW it turns out that "add documentation for two methods" really meant "add documentation for the entire ContiguousBlock class"...)